vhd/packer/install-dependencies.sh

```diff
@@ -374,6 +374,7 @@ K8S_VERSIONS="
 1.19.7
 1.19.6
 1.18.15
+1.18.15-azs
 1.18.14
 1.18.10-azs
```

Comment:
should this line go away?

Reply:
dont you mean 1.18.10-azs? If so yes, missed it.

Reply:
removed  1.18.10-azs